File: Backend-POS/POS.Main/POS.Main.Business.Admin/Models/Dashboard/TopSellingResponseModel.cs
namespace POS.Main.Business.Admin.Models.Dashboard;

public class TopSellingResponseModel
{
    public List<TopSellingItemModel> Food { get; set; } = new();
    public List<TopSellingItemModel> Beverage { get; set; } = new();
    public List<TopSellingItemModel> Dessert { get; set; } = new();
}

public class TopSellingItemModel
{
    public int MenuId { get; set; }
    public string MenuName { get; set; } = string.Empty;
    public int TotalQuantity { get; set; }
}

File: Frontend-POS/RBMS-POS-Client/src/app/core/api/models/top-selling-response-model.ts
/* tslint:disable */
/* eslint-disable */
import { TopSellingItemModel } from '../models/top-selling-item-model';
export interface TopSellingResponseModel {
  beverage?: Array<TopSellingItemModel> | null;
  dessert?: Array<TopSellingItemModel> | null;
  food?: Array<TopSellingItemModel> | null;
}
